components/searchwithin/metadata/searchwithin.yml

```diff
@@ -1,22 +1,7 @@
 name: Search within
+status: Deprecated
+deprecationNotice: Use a <a href="search.html">search field</a> with a separate control to filter the search instead.
```

Comment:
Let's incorporate the fix you found to add the spectrum link class manually here. We'll improve that in the 11ty implementation though so we won't have to remember this in future.